besspin/cyberPhys/ignition/canrepl.py

```diff
@@ -37,35 +37,19 @@
 
 # argument parser
 parser = argparse.ArgumentParser(description="Can ID Canlib REPL")
-parser.add_argument("-port", type=int, default=5030, help="IP of TCP Bus")
+parser.add_argument("-port", type=int, default=5002, help="IP of TCP Bus")
```

Comment:
We need to be able to switch between UDP and TCP. I use `canrepl` to send C&C messages for testing the ignition state machine.

Reply:
Cool. That is reasonable.